### ISSUE TITLE
DEM: Investigate replacing ALOS with FABDEM - a DEM rather than DSM

### ISSUE THREAD

**Issue by thejollygeo** [Blue-Ventures-Conservation/GEM#35]
FABDEM is a DEM that is truly a DEM at ground level, with forests removed, rather than ALOS and SRTM that are DSMs.
https://www.geographyrealm.com/fabdem-30-meter-resolution-bare-earth-dem-from-fathom/ 

On awesome GEE:
https://gee-community-catalog.org/projects/fabdem/

Do a test to see if FABDEM performs better over TAB and VEL (there were instances of TF being masked out by ALOS due to tree height)
- ask Court what the ALOS derived limits are over TAB and VEL
- do a QGIS comparison looking at high-res imagery

**Comment by thejollygeo** [comment]
An example where forest cover means an area is masked out is in TAB -13.704007,48.409933 where cropland is not masked out, but adjacent forested areas on either side are.  What does the FABDEM mask look like in this area?

![image](https://github.com/user-attachments/assets/f096dc8e-fb7f-4767-8786-3430ce14ffee)

**Comment by thejollygeo** [comment]
Example of how the ALOS DSM erroneously (and quite randomly) masks out some mangrove over Somone - would be interesting to see whether FABDEM is any better

![Image](https://github.com/user-attachments/assets/8c109267-8e5a-426b-8457-712caa82d423)

**Comment by thejollygeo** [comment]
@zibnix another DEM that I encountered back in Jun 2025!  Are you able to add FABDEM to the mask comparison script you've put together?

**Comment by zibnix** [comment]
Absolutely! This is now available to compare against our other approaches in the comparison scipt:
https://code.earthengine.google.com/?scriptPath=users%2Fcourtlandfowler%2Ftest%3Amask_comparisons
